package.json

```diff
@@ -92,6 +92,7 @@
     "react-addons-css-transition-group": "^15.4.1",
     "react-addons-shallow-compare": "^15.4.1",
     "react-addons-test-utils": "^15.4.1",
+    "react-burger-menu": "^2.1.4",
```

Comment:
`yarn add` is now the way to do it, to ensure that we have the version locking features that yarn provides.